mvt/ios/modules/base.py

```diff
@@ -187,11 +187,16 @@ def _find_ios_database(
                     for found_path in self._get_fs_files_from_patterns(root_paths):
                         file_path = found_path
                         break
+            # If we do not find any, we fail.
+            if file_path:
+                self.file_path = file_path
+            else:
+                raise DatabaseNotFoundError("unable to find the module's database file")
 
-        # If we do not find any, we fail.
-        if file_path:
-            self.file_path = file_path
+        # If there was an explicit file path specified, check if it actually exists.
         else:
-            raise DatabaseNotFoundError("unable to find the module's database file")
+            # If it doesn't exist, we fail.
+            if not os.path.exists(self.file_path):
+                raise DatabaseNotFoundError("unable to find the module's database file")
```

Comment:
I was experiencing an error when testing the new Viber module that caused the script to always fail whenever I provided an explicit `self.file_path` for the sqlite database. That was happening because when `self.file_path` was provided, the first `if` block would be skipped, moving on to the next `if/else` block which raised an error if `file_path` didn't exist, even if self.file_path did exist. These updates fix that error